### PR TITLE
Add details of kkollman's talk

### PR DESCRIPTION
**Pull request by inglesp**

### CODE REVIEW

File: sessions/talks/taking-the-robot-out-of-mr-robot-or-why-tech-sucks-how-you-can-help-to-make-it-better.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+---
+Not everything about the tech industry is as "cool" as it is sometimes made out
+to be – by the media, representations in fiction or indeed members of the tech
+community themselves – though not everyone is equally aware of how or why it is
+more difficult "to tech" for some than it is for others, regardless of innate
+talent or skills.
+
+In my talk I want to point out some of the barriers which make it harder or
+less fun for some of us to participate and how everyone can contribute to
+making tech a little more welcoming for everyone else. (& I might use examples
```
Comment (alexwlchan) [inline sessions/talks/taking-the-robot-out-of-mr-robot-or-why-tech-sucks-how-you-can-help-to-make-it-better.md:13]: Nitpick: sentences shouldn’t start with an ampersand, should use "and".

**Comment by alexwlchan** [comment]
I might push some commits with fixes for my nitpicks later, but I don't have access to Git right now.

File: sessions/talks/taking-the-robot-out-of-mr-robot-or-why-tech-sucks-how-you-can-help-to-make-it-better.md (hunk 1 above)
Comment (inglesp) [reply sessions/talks/taking-the-robot-out-of-mr-robot-or-why-tech-sucks-how-you-can-help-to-make-it-better.md:13]: I think it's OK for speakers to use non-standard grammar as long as the meaning's clear.

Comment (inglesp) [reply sessions/talks/taking-the-robot-out-of-mr-robot-or-why-tech-sucks-how-you-can-help-to-make-it-better.md:13]: (That said, there are some cases where we could think about making fixes, but this is borderline.)

Comment (alexwlchan) [reply sessions/talks/taking-the-robot-out-of-mr-robot-or-why-tech-sucks-how-you-can-help-to-make-it-better.md:13]: K, happy with it if you are.

**Pull request merged**